backend/Squirrel.Core/Squirrel.Core.WebAPI/Program.cs

```diff
@@ -1,6 +1,6 @@
 using Squirrel.Core.BLL.Extensions;
 using Squirrel.Core.WebAPI.Extensions;
-using Squirrel.Core.WebAPI.Middlewares;
+using Squirrel.ExceptionHandling.Middleware;
```

Comment:
Please, name this project  as Shared or something like this (maybe we even have it)
It will have more than just exceptions

Reply:
renamed to Shared